Godeps/_workspace/src/k8s.io/kubernetes/pkg/apiserver/handlers_test.go

```diff
@@ -41,13 +42,21 @@ func (f fakeRL) CanAccept() bool { return bool(f) }
 func (f fakeRL) Accept()         {}
 
 func expectHTTP(url string, code int, t *testing.T) {
-	r, err := http.Get(url)
+	var response *http.Response
+	err := wait.Poll(10, time.Duration(100*time.Millisecond), func() (bool, error) {
+		var e error
+		response, e = http.Get(url)
+		if response.StatusCode == 429 {
```

Comment:
Based on the error message, I'm actually not convinced this is correct.  `expectHTTP` passes a code.  If they wanted to accept a 429, they'd've passed it.  I can't see exactly where it failed, but based on the failing test, they expected a 200 and didn't get it.  That suggests to me that this `TestMaxInFlight` had a legit failure or at least a race in which they didn't properly predict the results.

The fact that the test itself talks about races concerns me.